embulk-output-sqlserver/src/test/java/org/embulk/output/sqlserver/SQLServerTests.java

```diff
@@ -46,31 +46,37 @@ public static void execute(String sql, String... options)
     {
         ConfigSource config = baseConfig();
 
-        ImmutableList.Builder<String> args = ImmutableList.builder();
-        args.add("sqlcmd")
-                .add("-U")
-                .add(config.get(String.class, "user"))
-                .add("-P")
-                .add(config.get(String.class, "password"))
-                .add("-H")
-                .add(config.get(String.class, "host"))
-                .add("-d")
-                .add(config.get(String.class, "database"))
-                .add("-Q")
-                .add(sql);
+        List<String> args = new ArrayList<>(Arrays.asList("sqlcmd",
+                "-U",
+                config.get(String.class, "user"),
+                "-P",
+                config.get(String.class, "password"),
+                "-H",
+                config.get(String.class, "host"),
+                "-d",
+                config.get(String.class, "database"),
+                "-Q",
+                sql
+                ));
+
         for (String option : options) {
-            args.add(option);
+            args.add((String)option);
         }
 
-        ProcessBuilder pb = new ProcessBuilder(args.build());
+        ProcessBuilder pb = new ProcessBuilder(Collections.unmodifiableList(args));
         pb.redirectErrorStream(true);
         int code;
         try {
             Process process = pb.start();
-            ByteStreams.copy(process.getInputStream(), System.out);
+            InputStream inputStream = process.getInputStream();
+            byte[] buffer = new byte[8192];
+            int readSize;
+            while ((readSize = inputStream.read(buffer)) != -1) {
+                System.out.write(buffer,0,readSize);
```

Comment:
The coding style convention prefers to have a whitespace (`' '`) after comma (`','`).